flybywiresim/aircraft
# feat(APU): Add APU start motor failure

Fixes #[issue_no]

## Summary of Changes

The goal of this PR is to add a failure to the APU start motor which, when active, prevents the APU from beeing started.

## Screenshots (if necessary)



## References

https://www.smartcockpit.com/docs/A320-Auxiliary_Power_Unit.pdf
This states the FAULT light illuminates for "no accelleration", so I dont know exactly if the FAULT light should come on or not in this case, an accuracy check would be highly appreciated.




## Additional context

This is my first contribution to this project, so dont judge me too hard.


Discord username (if different from GitHub): pizza187

## Testing instructions

1. Spawn in
2. Energize the aircraft
3. Observe the APU starting without the failure selected / Observe the APU not starting with the failure selected
4. Observe the N value drop back down when the failure is selected below 55% during the start cycle
5. When the failure is deselcted the FAULT light should disappear when the APU master switch is deselected and power to the electronic control box is cut
6. Test this in combination with the APU fire pushbutton


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
